### ISSUE TITLE
Création d'une adresse e-mail pour le groupe local de contributeur seine-eure

### ISSUE THREAD

**Issue by lefuturiste** [osm-fr/infrastructure#488]
Bonjour, 

Nous les membres du [groupe local Seine-Eure](https://wiki.openstreetmap.org/w/index.php?title=Groupe_Local_Seine-Eure) aimerions disposer d'une adresse email en `@openstreetmap.fr` afin d'avoir un point de contact pour le groupe local de contributeur et d'utilisateurs et pouvoir contacter les elus locaux, les journaux, faire la promotion du projet etc.

Je ne sais pas si ce genre de requête est courante, cela peut-il se faire ?

J'aurait pensé une adresse de type : `groupe.seine-eure@openstreetmap.fr`

Bonne journée

**Comment by lefuturiste** [comment]
> Ça doit être la première requête de ce genre :)
> 
> Du coup, ça serait juste un email pour envoyer des emails, avec redirection, sans liste email associée?

Ça serais un compte pour avoir des correspondance entre des personnes extérieures et les membres du groupe Seine-Eure (en privé), je pense que le plus simple pour l'instant est d'avoir une boite partagé ou on peut se logger.
Je ne sais pas ce que tu entend par "redirection", c'est pour forward les messages à des membres sur leur addr perso ?

Pour ce qui est d'une liste email, je ne sais pas encore si c'est utile mais pourquoi pas en rajouter.

**Comment by jocelynj** [comment]
Oui, une redirection, ça serait que tous les emails envoyés sur cette adresse @openstreetmap.fr serait redirigé vers d'autres adresses.

On n'a pas de boîte email partagé, uniquement des listes sur https://listes.openstreetmap.fr, et des redirections. J'ai l'impression que la liste correspondrait plus à ton besoin.

**Comment by vinber** [comment]
à voir où cela en est et peut-être reprendre la question en CA, qu'est ce que cela peut avoir comme impact les adresses en @openstreetmap.fr et qu'est ce que cela engage ? puisque cela signifie utiliser "l'image" de l'association :)

il me semble que quelques éléments avaient été discuté

**Comment by Cyrille37** [comment]
Bonjour @lefuturiste 
As-tu du nouveau pour ce ticket ?

**Comment by Cyrille37** [comment]
J'ai poké @lefuturiste là https://wiki.openstreetmap.org/wiki/User_talk:Lefuturiste#Cr%C3%A9ation_d%27une_adresse_e-mail_openstreetmap.fr

**Comment by lefuturiste** [comment]
Le besoin est toujours là.
Voici mes points :
- avoir une adresse email pour démarcher ou organiser des évènements au niveau local avec le nom de l'asso et pas avec mon adresse perso.
- avoir une liste mail, pour prévenir et organiser les évènements locaux.
